resources/helm/v2.1/istio-control/istio-discovery/templates/_affinity.tpl

```diff
@@ -0,0 +1,93 @@
+{{/* affinity - https://kubernetes.io/docs/concepts/configuration/assign-pod-node/ */}}
```

Comment:
you'll need to move this file into the `overlays` directory (instead of 2.1) and then run `make gen`, which will copy it here